src/test/java/com/amazon/opendistroforelasticsearch/sql/unittest/NestedFieldRewriterTest.java

```diff
@@ -58,47 +58,47 @@ public void selectWithoutFrom() {
     public void selectAll() {
         same(
             query("SELECT * FROM team t, t.employees"),
-            query("SELECT *, nested(employees.*) FROM team")
+            query("SELECT *, nested(employees.*, 'employees') FROM team")
```

Comment:
do we still support "SELECT *, nested(employees.*) FROM team"? If yes, do we need to have test cases for it?

Reply:
We don't explicitly support this syntax but the SQL++ one, but it still works. These tests checks whether the new AST generated from NestedRewriter matches the AST generated by explicitly passing nested function.